pkgs/development/compilers/bluespec/default.nix

```diff
@@ -1,66 +1,29 @@
-{ lib, stdenv
-, fetchFromGitHub
-, fetchpatch
-, autoconf
-, automake
-, fontconfig
-, gmp-static
-, gperf
-, libX11
-, libpoly
-, perl
-, flex
-, bison
-, pkg-config
-, itktcl
-, incrtcl
-, tcl
-, tk
-, verilog
-, xorg
-, yices
-, zlib
-, ghc
+{ lib, stdenv, fetchFromGitHub, fetchpatch, autoconf, automake, fontconfig
+, gmp-static, gperf, libX11, libpoly, perl, flex, bison, pkg-config, itktcl
+, incrtcl, tcl, tk, verilog, xorg, yices, zlib, ghc, asciidoctor, tex # docs
```

Comment:
Please revert this. It will get formatted like it was before in the future anyway.